nodejs/README.adoc

```diff
@@ -0,0 +1,341 @@
+.Guided steps to define topic details
+image::sak-create-topic.png[Image of wizard to create a topic]
+
+* *Topic name*: Enter `countries` as the topic name.
+* *Partitions*: Set the number of partitions for this topic. This example sets the partition to `1` for a single partition. Partitions are distinct lists of messages within a topic and enable parts of a topic to be distributed over multiple brokers in the cluster. A topic can contain one or more partitions, enabling producer and consumer loads to be scaled.
+* *Message retention*: Set the message retention time and size to the relevant value and increment. This example sets the retention time to `7 days` and the retention size to `Unlimited`. Message retention time is the amount of time that messages are retained in a topic before they are deleted or compacted, depending on the cleanup policy. Retention size is the maximum total size of all log segments in a partition before they are deleted or compacted.
+* *Replicas*: For this release of {product}, the replicas are preconfigured. The number of partition replicas for the topic is set to `3` and the minimum number of follower replicas that must be in sync with a partition leader is set to `2`. Replicas are copies of partitions in a topic. Partition replicas are distributed over multiple brokers in the cluster to ensure topic availability if a broker fails. When a follower replica is in sync with a partition leader, the follower replica can become the new partition leader if needed.
++
+After you complete the topic setup, the new Kafka topic is listed in the topics table for your Kafka instance. You can now run the Quarkus application to start producing and consuming messages to and from this topic.
+
+.Verification
+ifdef::qs[]
+* Is the `countries` topic listed in the topics table?
+endif::[]
+ifndef::qs[]
+* Verify that the `countries` topic is listed in the topics table.
+endif::[]
+
+[id="proc-running-nodejs-example-application_{context}"]
+== Running the Node.js example application
+
+After you configure your Node.js application to connect to a Kafka instance and you create the required Kafka topic, you can run the Node.js application to start producing and consuming messages.
+
+In this task, you'll use the following components of the Node.js application to produce and consume messages:
+
+* A `producer-backend` component that generates random country names and sends these names to the Kafka topic.
+* A `consumer-backend` component that consumes the country names from the Kafka topic.
+
+.Prerequisites
+* You've configured the Node.js example application to connect to the Kafka instance.
+* You've created the `countries` example Kafka topic.
+
+.Procedure
+. On the command line, navigate to the `reactive-example` directory corresponding to the repository that you cloned.
++
+.Navigating to the reactive-example directory
+[source]
+----
+$ cd ~/reactive-example
+----
+
+. Check out the `kafkajs` branch of the repository.
```

Comment:
I think this is the default branch right now when we checkout source code is it might not be needed.

WDYT @mhdawson ?

Ref:
https://github.com/nodeshift-starters/reactive-example

Reply:
I had just changed that recently I think @jbyrne-redhat was planning to remove it.

Reply:
Yes, given that `kafkajs` is now the default branch, I've removed the step that explicitly instructs users to check it out.